TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/Compiler.php

```diff
@@ -141,6 +141,9 @@ public function getProxyClass($fullClassName)
         if (in_array(substr($fullClassName, 0, 14), $this->blacklistedSubPackages)) {
             return false;
         }
+        if ($classReflection->isFinal() && preg_match('/^\s?\*\s?\@Annotation\s/m', $classReflection->getDocComment()) === 1) {
```

Comment:
>  I definitely prefer the proxy false solution then, also this might have impact on other peoples packages then

But that was exactly my reasoning: If we suddenly require a `Proxy(false)` annotation that change might be breaking for 3rd parties (i.e. https://github.com/Flowpack/jobqueue-common/blob/master/Classes/Annotations/Defer.php).
A different approach might be to copy *all* of the classes doc block to the proxy class!?